components/data-bridge/org.wso2.carbon.databridge.receiver.binary/src/main/java/org/wso2/carbon/databridge/receiver/binary/internal/BinaryDataReceiverServiceComponent.java

```diff
@@ -60,13 +58,10 @@ protected void activate(ComponentContext context) {
         }
         binaryDataReceiver = new BinaryDataReceiver(new BinaryDataReceiverConfiguration(dataBridgeReceiverService
                 .getInitialConfig()), dataBridgeReceiverService);
-        try {
-            binaryDataReceiver.start();
-        } catch (IOException e) {
-            log.error("Error while starting binary data receiver ", e);
-        } catch (DataBridgeException e) {
-            log.error("Error while starting binary data receiver ", e);
-        }
+                
+        context.getBundleContext().registerService(ServerStartupObserver.class.getName(), binaryDataReceiver, null);
+
```

Comment:
We can remove these unnecessary spaces.

Reply:
Fixed via https://github.com/wso2/carbon-analytics-common/pull/760/commits/9b9b85d1c3a947bf63b15231609c0144a5622461